rllib/core/testing/tests/test_bc_algorithm.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+import ray
+from ray.rllib.core.testing.torch.bc_module import (
+    DiscreteBCTorchModule,
+    # BCTorchRLModuleWithSharedGlobalEncoder,
+)
+from ray.rllib.core.testing.tf.bc_module import DiscreteBCTFModule
+
+from ray.rllib.core.testing.bc_algorithm import BCConfigTest
+from ray.rllib.utils.test_utils import framework_iterator
+
+
+class TestRLTrainer(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        ray.init()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        ray.shutdown()
+
+    def test_bc_algorithm(self):
+
+        config = (
+            BCConfigTest()
+            .rl_module(_enable_rl_module_api=True)
+            .training(_enable_rl_trainer_api=True, model={"fcnet_hiddens": [32, 32]})
+        )
+
+        # TODO (Kourosh): Add tf2 support
+        for fw in framework_iterator(config, frameworks=("torch")):
+            algo = config.build(env="CartPole-v1")
+            policy = algo.get_policy()
+            rl_module = policy.model
+
+            if fw == "torch":
+                assert isinstance(rl_module, DiscreteBCTorchModule)
+            elif fw == "tf":
+                assert isinstance(rl_module, DiscreteBCTFModule)
+
+    def test_bc_algorithm_w_custom_marl_module(self):
```

Comment:
just get rid of the test for now?

Reply:
I'm gonna fill out the test since it's relevant to this PR. Basically it tests whether this PR was effective.